src/components/Plans/PlanCalculator/reducers/plansByTypeReducer.test.js

```diff
@@ -85,6 +85,43 @@ describe('plansByTypeReducer', () => {
     });
   });
 
+  it(`${PLANS_BY_TYPE_ACTIONS.SEARCH_DISCOUNTS_BY_INDEX_PLAN} action`, () => {
```

Comment:
This name is not so descriptive. Try to make clear who, when, and expected.

Reply:
For example _it (plansByTypeReducer) **should** keep the previous index **when** SEARCH_DISCOUNTS_BY_INDEX_PLAN_ (I am not sure to understand it yet)